src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPartnerInfo.tsx

```diff
@@ -39,6 +39,10 @@ export class ArtworkSidebarPartnerInfo extends React.Component<
 
   render() {
     const { artwork } = this.props
+    const { collecting_institution } = artwork
```

Comment:
Oh @mzikherman - I think the latest on that was to display partner info for those works and not the collecting institution field. I believe that was intentional departure from what is currently displayed on artwork page.

There were pretty long discussions about that and I think the idea is that we want to encourage to correctly convert this 'collecting institution' field to proper availability.

Following this discussion this collecting_institution field was removed:
https://github.com/artsy/reaction/pull/1565

Wish we could tag Steven here but not sure about his github login:)

Reply:
Doh! I wish that were reflected in the comps I was going off of :(

Reply:
Yea, those keep changing and not always one to one with the latest accepted decisions. It is confusing though. Might be worth bringing at retro that when those things change - at least slap something like 'outdated' to the design spec.